support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.jsx

```diff
@@ -109,171 +109,194 @@ type CardFieldState =
 
 type CardFieldName = 'CardNumber' | 'Expiry' | 'CVC';
 
-type StateTypes = {
-  [CardFieldName]: CardFieldState,
-  currentlySelected: CardFieldName | null,
-};
-
-const fieldStyleControl = {
+const fieldStyle = {
   base: {
     fontFamily: '\'Guardian Text Sans Web\', \'Helvetica Neue\', Helvetica, Arial, \'Lucida Grande\', sans-serif',
-    fontSize: '16px',
     '::placeholder': {
       color: '#999999',
     },
-    lineHeight: '24px',
-  },
-};
-
-const fieldStyleDS = {
-  base: {
-    ...fieldStyleControl,
     fontSize: '17px',
     lineHeight: 1.5,
   },
 };
 
 const renderVerificationCopy = (countryGroupId: CountryGroupId, contributionType: ContributionType) => {
   trackComponentLoad(`recaptchaV2-verification-warning-${countryGroupId}-${contributionType}-loaded`);
-  return (<InlineError>Please tick to verify you&apos;re a human</InlineError>);
+  return (<div className="form__error"> {'Please tick to verify you\'re a human'} </div>);
 };
 
-
 const errorMessageFromState = (state: CardFieldState): string | null =>
   (state.name === 'Error' ? state.errorMessage : null);
 
-class CardForm extends Component<PropTypes, StateTypes> {
-
-  constructor(props: PropTypes) {
-    super(props);
+// Hook for monitoring the previous state of a prop
+const usePrevious = (value) => {
+  const ref = React.useRef();
```

Comment:
```suggestion
  const ref = useRef();
```
And the same for all the following usages